Adding Groovy plugin to Gradle project fails quarkusDev

When adding the Groovy plugin like:

```
plugins {
    id "java"
    id "groovy"
    id "io.quarkus"
}
```

then `quarkusDev` fails with:

```
❯ ./gradlew clean quarkusDev

> Task :compileJava
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

> Task :quarkusDev FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Some problems were found with the configuration of task ':quarkusDev' (type 'QuarkusDev').
> Directory '/Users/marceloverdijk/workspace/demo-backend-quarkus/build/classes/groovy/main' specified for property 'workingDir' does not exist.
> Directory '/Users/marceloverdijk/workspace/demo-backend-quarkus/src/main/groovy' specified for property 'sourceDir' does not exist.
```

It seems the `src/main/groovy` folder is expected.

I'm only adding Groovy to run Spock tests from `/src/test/groovy`.

When creating an empty `src/main/groovy` folder it keeps complaining about `build/classes/groovy/main`.

--- a/devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPluginExtension.java
+++ b/devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPluginExtension.java
@@ -108,7 +108,9 @@ public class QuarkusPluginExtension {
     private File getLastFile(FileCollection fileCollection) {
         File result = null;
         for (File f : fileCollection) {
-            result = f;
+            if (result == null || f.exists()) {
+                result = f;
+            }
         }
         return result;
     }

